src/view/screens/DebugNew.tsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react'
+import {View, Text as RNText} from 'react-native'
+import {CenteredView, ScrollView} from '#/view/com/util/Views'
+
+import {useSetColorMode} from '#/state/shell'
+import {
+  useBreakpoints,
+  useStyle,
+  useStyles,
+  Box,
+  Text,
+  Button,
+  H1,
+  H2,
+  H3,
+  H4,
+  H5,
+  H6,
+} from '#/view/nova'
+
+function ThemeSelector() {
+  const setColorMode = useSetColorMode()
+
+  return (
+    <Box row gap="s">
+      <Button onPress={() => setColorMode('system')}>
+        <Text>System</Text>
+      </Button>
+      <Button onPress={() => setColorMode('light')}>
+        <Text>Light</Text>
+      </Button>
+      <Button onPress={() => setColorMode('dark')}>
+        <Text>Dark</Text>
+      </Button>
+    </Box>
+  )
+}
+
+function BreakpointDebugger() {
+  const breakpoints = useBreakpoints()
+
+  return (
+    <Box>
+      <H3 pb="s" fontWeight="bold">
+        Breakpoint Debugger
+      </H3>
+      <Text pa="m" bg="l1" fontFamily="monospace">
+        {JSON.stringify(breakpoints, null, 2)}
+      </Text>
+    </Box>
+  )
+}
+
+function Hooks() {
+  const outer = useStyle({
+    pa: 'm',
+    bg: 'l1',
+  })
+  const {heading} = useStyles({
+    heading: {
+      color: 'l7',
+      fontSize: 'l',
+      fontWeight: 'bold',
+      gtTablet: {
+        fontSize: 'xl',
+      },
+    },
+  })
```

Comment:
Both direct usage and `StyleSheet`-like usage. Note responsive `gtTablet`. All responsive is `min-width` based, so base is mobile, `gtMobile` applies mobile and tablet, etc.